extensions/admin-tool-ext-win/src/main.ts

```diff
@@ -258,5 +246,17 @@ export async function buildUrn(serverName: string, node: azdata.objectexplorer.O
 		}
 		node = await node.getParent();
 	}
+
+	serverName = /\.database\.windows\.net$/i.test(serverName) ? serverName.slice(0, serverName.length - 21) : serverName;
 	return [`Server[@Name='${doubleEscapeSingleQuotes(serverName)}']`].concat(urnNodes).join('/');
+}
+
+function showStatusBarItem(text: string) {
+	statusBarItem.text = text;
+	statusBarItem.tooltip = text;
```

Comment:
isn't there a static  method to show a status bar message for a fixed amount of time, or do extensions not have access to it? The command line service uses such a method.

Reply:
Yeah there is. I went with this approach since using setStatusBarMessage means that if anyone else calls it the message will be overwritten. This implementation makes it so there's a separate item for this so it'll always show the full duration.

But after thinking about it that doesn't seem like a big enough concern to warrant the extra code - if people are opening dialogs it's unlikely they're doing much else anyways. And even if the message is overwritten it's really not that big of a deal since this is mostly just for immediate feedback anyways.